src/js/forecast.js

```diff
@@ -0,0 +1,35 @@
+const tempForecastDay1 = document.getElementById('temp-forecast-day1');
+const tempForecastDay2 = document.getElementById('temp-forecast-day2');
+const tempForecastDay3 = document.getElementById('temp-forecast-day3');
+const iconDay1 = document.getElementById('icon-day1');
+const iconDay2 = document.getElementById('icon-day2');
+const iconDay3 = document.getElementById('icon-day3');
+
+function getForecast() {
+  if (localStorage.getItem('location') !== null) {
```

Comment:
You can compare to null - but it is not mandatory. 
Any falsy values will be "false"
https://developer.mozilla.org/en-US/docs/Glossary/Falsy

So this is also a valid thing:
```
if(localStorage.getItem('location)) {
...
}```

Reply:
yes, agree. Fixed.